src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -34,8 +35,11 @@ public class AddressBook implements ReadOnlyAddressBook {
         persons = new UniquePersonList();
         transactions = new UniqueTransactionList();
     }
+    private Comparator<Person> currComparator;
```

Comment:
Maybe change the variable name to `currentPersonComparator` or `personComparator`